components/layout/model.rs

```diff
@@ -436,6 +436,21 @@ impl MaybeAuto {
     }
 }
 
+// Return used value for inline-size or block-size.
```

Comment:
Can you make this a doc comment? Also, this doesn't return the used value (the used value is technically the _final_ pixel value).

Probably you can reword it somehow. This returns a middle-ground between the computed value and the used value.